ext/js/language/language-util.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2024  Yomitan Authors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/** @type {import('language').TextPreprocessorOptions<boolean>} */
+export const basicTextPreprocessorOptions = [false, true];
```

Comment:
🤔 i don't see the intent behind moving `basicTextPreprocessorOptions` out of `text-preprocessors.js`

Reply:
#In my mind `text-preprocessor` files only export `TextPreprocessor`s, but either way is fine.

Reply:
tbh it may even be overkill to use a shared 2-tuple rather than just re-declaring it, but I don't feel strongly about any of this.